core/src/banking_stage/transaction_scheduler/transaction_state_container.rs

```diff
@@ -170,6 +171,13 @@ impl TransactionStateContainer {
             .remove(id)
             .expect("transaction must exist");
     }
+
+    pub(crate) fn get_min_max_prioritization_fees(&self) -> MinMaxResult<u64> {
+        self.priority_queue
+            .iter()
+            .map(|priorities| priorities.priority)
+            .minmax()
```

Comment:
This would iterate over every element, and take O(n) time. That's not worth the cost just to collect a metric.
It's a minmax heap, just peek the min and max, then it's O(1).

Reply:
Duh yeah for sure, i made the same thing before don't know why I did it like this here.